.gitignore

```diff
@@ -127,3 +127,4 @@ log
 logs/states/trading_state_history.json
 backtesting/data/
 cython_debug/
+user/tentacles_config/
```

Comment:
I think we should just ignore `user` (by removing the `config.json` ignore above and the `tentacles_config/`).

Reply:
right, it's better